### ISSUE TITLE
A11Y: Analytics form is missing labels on form controls

### ISSUE THREAD

**Issue by jannakha** [apigee/apigee-edge-drupal#486]
## Description
Accessibility check fails App/Analytics form:

WCAG 1.3.1, WCAG 4.1.2
path: #edit-metrics, #edit-quick-date-picker

Fix ONE of the following:
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- Element has no title attribute or the title attribute is empty

![image](https://user-images.githubusercontent.com/1356783/92988611-7b970c80-f510-11ea-92f2-fe15e75e187a.png)

**Comment by arlina-espinoza** [comment]
Fixed by #503 .
